kymatio/scattering2d/backend/tensorflow_backend.py

```diff
@@ -0,0 +1,176 @@
+# Authors: Edouard Oyallon, Sergey Zagoruyko
+
+import tensorflow as tf
+import numpy as np
+from collections import namedtuple
+
+BACKEND_NAME = 'tensorflow'
+
+def _iscomplex(x):
+    return x.dtype == np.complex64 or x.dtype == np.complex128
```

Comment:
When Tensorflow tensors are 'evaluated', they are automatically converted into numpy arrays. So this is fine.

Reply:
Also, I tried `tf.complex64` and `tf.complex128` and it just error ed out. I'll try again tomorrow to make sure, but I am very sure it didn't work.

Reply:
So I think this function is not worth here because indeed, tensorflow will throw quickly an error.

Reply:
It's mostly for compatibility. Once we implement the skcuda `cdgmm`, we need these safeguards to avoid segfaults.

Reply:
ok.